### ISSUE TITLE
UnconfuserEx

### ISSUE THREAD

**Issue by cawk** [yck1509/ConfuserEx#381]
hello i am trying to modify confuserex 0.6.0 what do i need to edit to make unconfuserex stop working 
i have removed all watermarks and still says file is unpacked successfully (the app isnt however) but i would still like it to fail unconfuserex

**Comment by yck1509** [comment]
Hi,
Please provide a link to the tool you used and I'll have a look.

**Comment by MyGameFrame** [comment]
UnConfuserEx v1.0
NoFuserEx v1.0

None of the two was not able to open 0.6.0. Tried with/without enabling the packer (compressor). Thanks

Anyway I am going to modify the source suggested by the author so at least they need to work harder to get my codes.

yck1509: https://pledgie.com/campaigns/25451 is the right for donation? I did donate few times $50 bucks if my memory servers me right but it's an odd link. I mean I am used to use direct PP links for donation ...

**Comment by yck1509** [comment]
Hi,
Most of the new protection options are implemented through parameters. If you are referring to the two tools you used, as far as I know, these two tools cannot deal with the newer protection options implemented in v0.5/0.6.

**Comment by cawk** [comment]
that is incorrect with 0.5/0.6 you can deob a file automatically with a selection of tools as long as its not modified

**Comment by yck1509** [comment]
Do you have a download link to these tools? AFAIK, UnConfuserEx only deals with Packers/Anti-Tamper but no other protections, and I could not find the 'NoFuzerEx' tool you refer to.

**Comment by cawk** [comment]
do you have skype or email so i can contact you there? as dont want to be giving out all tools on here

**Comment by yck1509** [comment]
You can contact me at confuser.net@gmail.com

**Comment by cawk** [comment]
okay sent you an email

**Comment by MyGameFrame** [comment]
NoFuserEx v1.0: https://www.nulled.cr/topic/115809-nofuserex-confuserex-deobfuscator/

Download: https://www.sendspace.com/file/gy6bip (the original link) It is verified already many times at virustotal.com

Maybe you want to try it with <= 0.5.0. It did not worked with 0.6.0. My suggestion for the author is to make all encryption algorithms dynamic. I mean a different bit shifting etc. every time you pack an assembly.

NoFuerEx claims to be able to also null control flow obfuscation. Everything needs to be eventually decrypted in the memory so basically this protection is the most powerful as it leave them spaghetti code. Really I wonder how one can parse it back to the original version?

**Comment by bill99930432** [comment]
@yck1509  @cawk i need help with de-obfuscating 0.6.0, could you help me? my email is: mistersmithanonymous2009 [at] hotmail[dot] com
